### PR TITLE
Commute distances

### PR DESCRIPTION
**Pull request by koffeinfrei**
Currently we can automatically calculate the commute distances from the Controllr data as we have the employee's address. I'm not sure though if the info as such is very meaningful. A commute distance in time would be more interesting I guess. But that info is very subjective and we'd have to store that info manually for every employee.
Thoughts?

panter ag dashboard 2016-02-15 16-57-12

### CODE REVIEW

**Comment by exul** [comment]
It might not be accurate (since it depends on how you get to work) but I think it's funny.

Btw. who lives 127.9km away from Colab? :scream: Because I already live far away and according to openstreetmap that's only 24km.

Since we get a growing number of figures we should think about splitting them into multiple boards and only have the "most important" ones on the front page.

:+1:

**Comment by koffeinfrei** [comment]
> Since we get a growing number of figures we should think about splitting them into multiple boards and only have the "most important" ones on the front page.

I'm thinking that with the rebranding we'll get new fonts and design possibilities which will allow us to make smaller cards and use smaller fonts. The current font only works in very large scale.

And yes, maybe we'll have to make multiple boards anyway...

File: lib/controllr.rb [hunk 1]
```diff
@@ -1,7 +1,11 @@
 require 'json'
+require 'geocoder'
+require 'redis'
 require './lib/age'
 
 class Controllr
+  OFFICE_ADDRESS = ENV['OFFICE_ADDRESS']
```
Comment (huerlisi) [inline lib/controllr.rb]: You could get the office address from controllr also: `/api/system_settings.json`. See http://controllr.panter.biz/system_settings

**Comment by huerlisi** [comment]
:+1:

File: lib/controllr.rb (hunk 1 above)
Comment (koffeinfrei) [inline lib/controllr.rb]: Updated.

**Pull request merged**